### ISSUE TITLE
Only first metadata is preserved

### ISSUE THREAD

**Issue by jsab** [clojure-emacs/clj-refactor.el#362]
## Expected behavior

`cljr-clean-ns` should keep all namespace metadata.

## Actual behavior

It only keeps the first piece of metadata.

## Steps to reproduce the problem

The following namespace:

```` clojure
(ns ^:a ^:b ^:c test-record.core
  "My beautiful ns."
  (:require [clojure.pprint :refer [pprint]]))
````

Becomes

```` clojure
(ns ^:a test-record.core
  "My beautiful ns.")
````

### clj-refactor.el version information

clj-refactor 2.3.0-SNAPSHOT (package: 20161223.1457), refactor-nrepl 2.3.0-SNAPSHOT

### CIDER version information

```
;; CIDER 0.15.0snapshot (package: 20170104.1330), nREPL 0.2.12
;; Clojure 1.8.0, Java 1.8.0_112
```

### Leiningen or Boot version

Leiningen 2.7.1 on Java 1.8.0_112 Java HotSpot(TM) 64-Bit Server VM

### Emacs version

GNU Emacs 25.1.1 (x86_64-apple-darwin16.3.0, NS appkit-1504.76 Version 10.12.2 (Build 16C67)) of 2017-01-06

### Operating system

macOS Sierra 10.12.2 (16C67)

**Comment by jsab** [comment]
Related issue: https://github.com/clojure-emacs/clj-refactor.el/issues/223

**Comment by benedekfazekas** [comment]
fixed with clojure-emacs/refactor-nrepl/abadad9c68d5888ef47ae679c74b53a6d39616cf

**Comment by benedekfazekas** [comment]
on clojars
